src/core.js

```diff
@@ -138,7 +138,7 @@ const Editable = module.exports = class Editable {
   disable (target) {
     const document = this.win.document
 
-    target = target ? [target] : Array.from(document.querySelectorAll(`.${config.editableClass}`))
+    target = target ? [target] : [...document.querySelectorAll(`.${config.editableClass}`)]
```

Comment:
We could also replace the `target.forEach` call with a `for (const element of target)` loop. That way this conversion won't be necessary at all.